packages/core/services/google-maps-types.ts

```diff
@@ -82,6 +82,13 @@ export interface MarkerLabel {
   text: string;
 }
 
+export interface MarkerIcon {
```

Comment:
if so, you need to export GoogleSymbol and MarkerIcon in root `map-types.js`

Reply:
Okay